tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -973,3 +975,87 @@ def test_negative_delete_by_name(self, content_view):
             ContentView.filter.delete(
                 {'content-view-id': content_view['id'], 'name': gen_string('utf8')}
             )
+
+    @pytest.mark.stream
+    @pytest.mark.tier2
+    def test_positive_check_filters_applied(self, target_sat, module_org, content_view):
+        """Ensure the applied filters are indicated and listed correctly in the CVV info.
+
+        :id: ab72af3f-6bee-4aa8-a74a-637fe9b7e34a
+
+        :steps:
+            1. Publish first CVV with no filters applied, assert no applied filters are indicated
+               nor listed.
+            2. Randomly add filters to the CV, publish new CVVs and assert that the applied filters
+               are indicated and the correct filters are listed.
+            3. Randomly remove filters from the CV, publish new CVVs and assert that the applied
+               filters are indicated when at least one filter exists and the correct filters were
+               removed.
+
+        :expectedresults:
+            1. Hammer shows correctly if a CVV has filter(s) applied, no matter the filter type,
+               count, nor order.
+            2. Hammer lists correctly the applied filter(s), no matter the filter type, count
+               nor order.
+        """
+        f_types = ['rpm', 'package_group', 'erratum', 'modulemd', 'docker']
+        filters_applied = []
+
+        # Publish first CVV with no filters applied
+        target_sat.cli.ContentView.publish({'id': content_view['id']})
+        cvv = target_sat.cli.ContentView.info({'id': content_view['id']})['versions'][0]
+        cvv_info = target_sat.cli.ContentView.version_info(
+            {'id': cvv['id'], 'include-applied-filters': 'true'}
+        )
+        assert cvv_info['has-applied-filters'] == 'no'
+        assert 'applied-filters' not in cvv_info
+
+        # Randomly add filters to the CV, assert correct CVV info values
+        for f_type in random.choices(f_types, k=random.randint(1, 5)):
+            cvf = target_sat.cli.ContentView.filter.create(
+                {
+                    'content-view-id': content_view['id'],
+                    'name': gen_string('alpha'),
+                    'organization-id': module_org.id,
+                    'type': f_type,
+                    'inclusion': random.choice(['true', 'false']),
+                },
+            )
+            filters_applied.append(cvf)
+
+            target_sat.cli.ContentView.publish({'id': content_view['id']})
+            cvv = max(
+                target_sat.cli.ContentView.info({'id': content_view['id']})['versions'],
+                key=lambda x: int(x['id']),
```

Comment:
Version with highest id? That's fishy. Comparing versions would make more sense and be future-proof. I think this will work in every reasonable scenario though. Not blocking.

Reply:
Yes, I need the latest CV version published. Normally I would go with something like `cv_info['versions'][-1]`, but there is a bug so we sometimes get the versions unsorted: https://bugzilla.redhat.com/show_bug.cgi?id=2209758

Reply:
Aaah, gotcha. It would have to be `float(x['version'])` and we would need to pray the versioning won't change from `X.Y` to `X.Y.Z` anytime soon. Making it less future-proof in my eyes.
```
'9.0' > '10.0'
Out[4]: True
float('9.0') > float('10.0')
Out[5]: False
```

Reply:
You sure can code it more generally. Comparing it as floats would be really ugly :)
But as I said, this is not blocking.